classes/Resizer.php

```diff
@@ -795,6 +799,14 @@ public function modify()
                     list($r, $g, $b) = explode(',', $value);
                     $this->im->colorize($r, $g, $b);
                     break;
+                case 'insert':
+                    $exp = explode(',', $value);
+                    $path = $exp[0];
+                    $position = (isset($exp[1]))? $exp[1] : null;
+                    $x = (isset($exp[2]))? $exp[2] : null;
+                    $y = (isset($exp[3]))? $exp[3] : null;
```

Comment:
@PubliAlex please add a space between the closing brackets and the question marks in the line above.

Reply:
Regex and space added :)